docs/reference/cluster/nodes-stats.asciidoc

```diff
@@ -2099,6 +2103,62 @@ Number of failed operations for the processor.
 =======
 ======
 
+[[cluster-nodes-stats-api-response-body-indexing-pressure]]
+`indexing_pressure`::
+(object)
+Contains indexing pressure statistics for the node.
```

Comment:
maybe cross-link to the indexing pressure section here (index-modules-indexing-pressure)